JASP-Desktop/po/jasp_nl.po

```diff
@@ -16078,10 +15845,2443 @@ msgctxt "RibbonModel|"
+
+msgctxt "auditBayesianEvaluation|"
+msgid "Correlation plot"
+msgstr "Correlatiediagram"
+
+msgctxt "auditBayesianEvaluation|"
+msgid "Prior and posterior"
+msgstr "Prior en posterior"
+
+msgctxt "auditBayesianEvaluation|"
+msgid "x-axis limit"
+msgstr "x-as limiet"
+
+msgctxt "auditBayesianEvaluation|"
+msgid "Additional info"
+msgstr "Aanvullende info"
+
+msgctxt "auditBayesianEvaluation|"
+msgid "Shade"
+msgstr "Markeer"
+
+msgctxt "auditBayesianEvaluation|"
+msgid "Credible region"
+msgstr "Geloofwaardigheidsgebied"
+
+msgctxt "auditBayesianEvaluation|"
+msgid "Support regions"
+msgstr "Ondersteuningsgebieden"
+
+msgctxt "auditBayesianEvaluation|"
+msgid "None"
+msgstr "Geen"
+
+msgctxt "auditBayesianEvaluation|"
+msgid "<b>Download Report</b>"
+msgstr "<b>Download verslag</b>"
+
+msgctxt "auditBayesianPlanning|"
+msgid "Prior Information"
+msgstr "Prior Informatie"
+
+msgctxt "auditBayesianWorkflow|"
+msgid "Prior Information"
+msgstr "Prior Informatie"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Population Materiality"
+msgstr "Populatiematerialiteit"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Absolute"
+msgstr "Absoluut"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Relative"
+msgstr "Relatief"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Population"
+msgstr "Populatie"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Size"
+msgstr "Grootte"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Value"
+msgstr "Waarde"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Audit Risk"
+msgstr "Audit Risico"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Confidence"
+msgstr "Betrouwbaarheid"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Record ID's"
+msgstr "Transactie ID's"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Audit Result"
+msgstr "Auditresultaat"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Book Values <i>(required)</i>"
+msgstr "Boekwaarden <i>(verplicht)</i>"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Book Values <i>(optional)</i>"
+msgstr "Boekwaarden <i>(optioneel)</i>"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Selection Counter <i>(optional)</i>"
+msgstr "Selectieteller <i>(optioneel)</i>"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Annotation Method"
+msgstr "Annotatiemethode"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Audit values"
+msgstr "Audit waarden"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Correct / Incorrect"
+msgstr "Correct / Incorrect"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Use summary statistics"
+msgstr "Gebruik samenvattende statistieken"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Sample size"
+msgstr "Steekproefgrootte"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Found errors"
+msgstr "Gevonden fouten"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Advanced Options"
+msgstr "Geavanceerde Opties"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Estimation Method"
+msgstr "Schattingsmethode"
+
+msgctxt "auditClassicalEvaluation|"
+msgid "Stringer"
+msgstr "Stringer"
```

Comment:
Is dit jargon?

Reply:
Komt van de Stringer bound, standaard methode hierin. Misschien `bound` er nog bij?